tests/test_cli.py

```diff
@@ -0,0 +1,215 @@
+#!/usr/bin/env python3
+
+from pytest import fixture
+from subprocess import Popen, PIPE
+
+
+@fixture
+def cli_args(shodan_key: str, censys_id: str, censys_secret: str) -> list:
+    """
+    Sets all needed arguments for execution and testing
+    :param shodan_key: shodan key value
+    :param censys_id: censys id value
+    :param censys_secret: censys secret value
+    :return: all constant arguments
+    """
+    sk = shodan_key
+    ci = censys_id
+    cs = censys_secret
+    base_args = [
+        "./grinder.py",
+        "-r",
+        "-u",
+        "-cu",
+        "-cp",
+        "-sk",
+        sk,
+        "-ci",
+        ci,
+        "-cs",
+        cs,
+        "-ni",
+        "-d",
+    ]
+    return base_args
+
+
+def test_cli_base_args(cli_args: list) -> None:
+    """
+    Execute base arguments and check results
+    :param cli_args: needed arguments for execution and testing
+    :return: None
+    """
+    p = Popen(["./grinder.py"], stdout=PIPE, stderr=PIPE)
```

Comment:
Put this into the variable like `GRINDER_RUN_COMMAND` or something like this, because you use it everywhere in code.